### PR TITLE
Apply example_ynh

### PR DESCRIPTION
**Pull request by yalh76**
## Solution

- *Apply example_ynh*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by yalh76** [comment]
!testme

File: conf/app.src [hunk 1]
```diff
@@ -1,6 +1,7 @@
 SOURCE_URL=https://github.com/pluxml/PluXml/archive/v5.8.8.zip
```
Comment (pp-r) [inline conf/app.src:1]: PluXml 5.8.8 zip seems to be unavailable unloke PluXml 5.8.9 that REVERT v5.8.8 FIX Static pages PHP injection vulnerability https://github.com/pluxml/PluXml/issues/558 (P3ter, Moritz Huppert)
SOURCE_URL=https://github.com/pluxml/PluXml/releases/download/5.8.9/pluxml-5.8.9.zip
e910d61041c56ce97085c0251f617ee2230ee7b7557e7af1c7cb513ce2d8b060

File: scripts/_common.sh [hunk 2]
```diff
@@ -4,10 +4,10 @@
 # COMMON VARIABLES
 #=================================================
 
-YNH_PHP_VERSION=7.4
+php_dependencies="php$YNH_DEFAULT_PHP_VERSION-gd php$YNH_DEFAULT_PHP_VERSION-xml"
 
-# dependencies used by the app
-extra_php_dependencies="php${YNH_PHP_VERSION}-gd php${YNH_PHP_VERSION}-xml"
+# dependencies used by the app (must be on a single line)
+pkg_dependencies="deb1 deb2 $php_dependencies"
```
[the remark this answers is not in the document]
Comment (yalh76) [reply scripts/_common.sh]: fixed ^^ thanks

File: conf/app.src (hunk 1 above)
Comment (yalh76) [reply conf/app.src]: also fixed, thanks

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
!testme

**Review by pp-r — APPROVED** [review]

**Pull request merged**